utils/config-utils.sh

```diff
@@ -26,18 +26,30 @@ config::set_config_digests() {
     local -r names=($(yq '.configs."*.*".name' "${DOCKER_COMPOSE_PATH}"))
     local -r composeFolderPath="${DOCKER_COMPOSE_PATH%/*}"
 
-    for ((i = 0; i < ${#files[@]}; i++)); do
-        file=${files[$i]}
-        name=${names[$i]}
+    if [[ "${files[*]}" != *"null"* ]] && [[ "${names[*]}" != *"null"* ]]; then
+        log info "Setting config digests"
 
-        fileName="${composeFolderPath}${file//\.\///}" # TODO: Throw an error if the file name is too long to allow for a unique enough digest
-        envVarName=$(echo "${name}" | grep -P -o "{.*:?err}" | sed 's/[{}]//g' | sed 's/:?err//g')
+        for ((i = 0; i < ${#files[@]}; i++)); do
+            file=${files[$i]}
+            name=${names[$i]}
 
-        # generate and truncate the digest to conform to the 64 character restriction on docker config names
-        envDeclarationCharacters=":?err" # '${:?err}' from setting an env variable
-        remainder=$((64 - (${#name} - ${#envVarName} - ${#envDeclarationCharacters})))
-        export "${envVarName}"="$(cksum "${fileName}" | awk '{print $1}' | cut -c -${remainder})"
-    done
+            fileName="${composeFolderPath}${file//\.\///}" # TODO: Throw an error if the file name is too long to allow for a unique enough digest
```

Comment:
Don't know if this is the PR where we should be doing it, but we said we were going to stick to google's style guide for bash, which prefers using snake_case

Reply:
yes, I will fix any function I go through when doing refactoring, fixed in [4d4b85c](https://github.com/jembi/platform/pull/188/commits/4d4b85c17604540e8d31869e0e0bc75f05133470)

Reply:
yes nice catch, fixed in [6353d0e](https://github.com/jembi/platform/pull/188/commits/6353d0e43eadde508585a5cd47c5954aa28d8a46)